### DISCUSSION TITLE
Assuming "information" is key -- perception is the other end, and it's huge! (I would say...)

### DISCUSSION THREAD

**Discussion by russellhanna** [OpenMined/courses#237] [Feedback]
I realize that there's a challenge trying to build a course about how **influence** works, instead of just how **information** works. Information is tractable. Influence is much more ...ethereal. But it's influence that is driving the problems you're starting out to address. It's influence that is at the end of all the streams of information you're outlining. Please don't skip over influence.

Facebook and YouTube (and TikTok!) are much discussed for their ability to bring content to feed and reinforce the biases people have. Few are the critical thinkers. Even fewer, I imagine, are those who get a notion and then try to _disprove_ themselves in order to validate their insight. Clearly information is kinda secondary to this process, since "information" is going to be filtered _out_ a lot of the time.

So in my mined (see what I did there? ;-) the focus needs to be shifted to the right a little bit -- onto the filtering mechanisms and the fact that selling ads is based around feeding appetites... like replacing all the fats with sugars in American processed foods. The incentives (making money by pandering to followers, mainly) matter a lot. I think this is different from addictive substances, though, because information by itself is not harmful. Filtering can feed or relieve ignorance, and ignorance is harmful. Information, by itself, breaks up against the blind spots people have. You can't ask if you're not aware to begin with.

But I don't care at all for a paternalistic view of "true" and "false" and "good for you." That's colonialism, right? (Or something like that.)

I started down the same path you are on, so I'm taking your course. But I've also found that natural language processing is really far away from anything like "understanding." We are a long way from being able to identify fake news. And I've read snippets that suggest it's not the "truthiness" of the information that matters. Instead, the salacious lies are _much more fun_ than the boring truth, even when both versions are immediately available. If the snippets are right, we need to talk about this filtering.

Bottom line, then: The filters and the perverse incentives are what I want to debug. I want you to quickly bring your focus forward in the ecosystem to look at how to make truth appealing or make lies unappealing, or how to help people spot their vulnerabilities to the tricks of their own amygdalas... or something like that. (If I knew the answer I'd have done it myself. Right?) 

But you've got me. I'm in. What's next?
